### DISCUSSION TITLE
Password field always encrypted

### DISCUSSION THREAD

**Discussion by lucabecchetti** [spatie/laravel-settings#128] [Q&A]
I have to encrypted fieldS: 

User and Password, i see them encrypted inside the database, and WHEN i use class from tinker, just user is in clear:

```
=> App\Models\Settings {#687
     +mail_username: "no-reply@isite.com",
     +mail_password: "$2y$10$a3k4HXyzQHaL6miMCdxJvOHfDEcxSQ/YP/BdvnFufuJR08gbIkhOG",
```

I need encrypted data when i use the class, why just one is encrypted?

`
    public ?string $mail_username;
    public ?string $mail_password;
    /**
     * @return string[]
     */
    public static function encrypted(): array
    {
        return [
            'mail_password',
            'mail_username'
        ];
    }
`

`
$this->migrator->addEncrypted('email.mail_username', null);
$this->migrator->addEncrypted('email.mail_password', null);
`

**Comment by rubenvanassche** [comment]
Hi @lucabecchetti,

I tried to reproduce but couldn't, both fields are unencrypted when required, could you provide me a reproduced version of this problem?
